presto-main/src/main/java/com/facebook/presto/testing/InterfaceTestUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.testing;
+
+import com.google.common.collect.ImmutableSet;
+
+import java.lang.reflect.Method;
+
+import static java.lang.String.format;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+public final class InterfaceTestUtils
+{
+    private InterfaceTestUtils() {}
+
+    // Note: this should be in sync with com.facebook.presto.spi.connector.classloader.TestClassLoaderSafeWrappers.assertAllMethodsOverridden
```

Comment:
Instead of copying the method, let's move the original method into a utility class in `com.facebook.presto.spi.testing`. Then we can add a `test-jar` dependency to use it (we already have that in `presto-main`).